app/actions/LoginActions.js

```diff
@@ -1,26 +1,31 @@
-import { client, getBalance } from './client';
-export const LOGIN = 'LOGIN';
-export const SET_CLIENT = 'SET_CLIENT';
+import { client } from '../middleware/grpc/client';
 
-export const login = (address, port, passphrase) => ({
-    address: address,
-    port: port,
-    passphrase: passphrase,
-    type: LOGIN
-})
+export const LOGIN_ATTEMPT = 'LOGIN_ATTEMPT';
+export const LOGGED_FAILED = 'LOGGED_FAILED';
+export const LOGGED_SUCCESSFULLY = 'LOGGED_SUCCESSFULLY';
```

Comment:
The getbalance constant uses _SUCCESS rather than _SUCCESSFULLY.  We should probably go with just one (I prefer the shorter one).